fractal-puppy/Engine
# Bug/Old Player sounds not working

Fixed:
- Skills sounds.
- Lowered enemies steps sound.
- Changed potions names.

To test:
- Make a BombDrop to check if it has sound.
- Check that the potions name is "Health Potion" or "Mana Potion".
